src/itemlistformaction.cpp

```diff
@@ -376,6 +377,108 @@ void ItemListFormAction::process_operation(Operation op,
 			v->show_error(_("Error: no item selected!"));
 		}
 	} break;
+	case OP_SAVEALL: {
+		LOG(Level::INFO,
+			"ItemListFormAction: saving all items");
+		if (visible_items.size() == 1) {
+			std::string directory = v->run_dirbrowser("");
+			if (directory.back() != '/') {
+				directory = directory + "/";
+			}
+
+			const std::string filename = v->get_filename_suggestion(visible_items[0].first->title());
+			const std::string fpath = directory + filename;
+
+			struct stat sbuf;
+			if (::stat(fpath.c_str(), &sbuf) != -1) {
+				std::string input_options = _("yn");
+				char c = v->confirm(strprintf::fmt(
+						_("Overwrite %s in %s? "
```

Comment:
Paths can contain spaces, so it's better to put them in funny quotes like ``` `these'```. Same in two messages below.

Reply:
I confess I did not understand very well why this is and what you are specifically referring to.

Reply:
Okay, let me explain with an example. Imagine I'm saving an article titled "random files" in a directory named "read asap!". if that file already exists, I'll get a message that says:

```
Overwrite random files.txt in read asap!? (y:Yes n:No)
```

It's not very clear where filename starts and ends. Same for the directory name.

So I suggest putting both in quotes:

```
Overwrite `random files.txt' in `read asap!`? (y:Yes n:No)
```

I believe this is more readable. Also, this is exactly what `FileBrowserFormAction` does with its confirmation messages, so if nothing else, we should do the same here for consistency. (Note that the first quote is not actually a quote, it's a backtick.)

Reply:
Done. :-)